main.go

```diff
@@ -64,6 +67,14 @@ func runMain() error {
 		return err
 	}
 
+	log.Println("checking if there is enough room on disk to record")
+	if enoughDiskSpace, err := diskSpaceCheck(conf.MinDiskSpace); err != nil || !enoughDiskSpace {
```

Comment:
The usual way to structure this is something like:

```go
if ok, err := diskSpaceCheck(...); err != nil {
    // handle err
} else if !ok {
    // handle flag
}
```

It's easier to read and understand that way.